pkgs/applications/misc/calibre/default.nix

```diff
@@ -71,7 +72,7 @@ stdenv.mkDerivation rec {
     pkg-config
     qmake
     removeReferencesTo
-    wrapQtAppsHook
```

Comment:
don't remove this `qtWrapperArgs` won't be set

Reply:
Mmh.. 
I looked at the original fix to the problem at https://github.com/NixOS/nixpkgs/pull/162089 and there wasn't any `wrapQtAppsHook` in `nativeBuildInputs` either. But I guess it doesn't hurt adding it again. (But just for reference: It works just fine without it..)

Reply:
maybe its required with the new version, try `XDG_DATA_DIRS= calibre`

Reply:
I don't quite understand..Shall I add a `--prefix XDG_DATA_DIRS=calibre` to `wrapProgram`? What would be the use for it?

Reply:
nah i mean run that command and if it doesn't work then some impurities are coming from your environment

Reply:
Overriding `XDG_DATA_DIRS` before starting calibre still results in a working program (just with a warning about missing icon) :
```
> result/bin/calibre

(python:111418): Gtk-WARNING **: 07:49:04.768: Could not find the icon 'user-home-symbolic-ltr'. The 'hicolor' theme
was not found either, perhaps you need to install it.
You can get a copy from:
        http://icon-theme.freedesktop.org/releases
```
Your question prompted me to question, whether we need the `wrapQtAppsHook` at all. Turns out, we don't. Calibre compiles and runs just fine without `wrapQtAppsHook`.